README.rst

```diff
@@ -48,8 +48,12 @@ now what you have to do looks like this:
 
 ``orion -n experiment_name ./run.py --mini-batch~'randint(32, 256)'``
 
-Check out our `pytorch example`_ for a quick demonstration!
+Check out our `getting started guide`_ or `this presentation
+<https://docs.google.com/presentation/d/18g7Q4xRuhMtcVbwmFwDfH7v9gKS252-laOi9HrEQ7a4/edit?usp=sharing>`_
```

Comment:
I'm not sure the slides are easy to follow without explanations. I tend to make minimalist slides without much text. Do we have access to the video recording outside of Mila instead?

Reply:
We do it's here: https://bluejeans.com/playback/s/4WUezzFCmb9StHzYgB0RjVbTUCKnRcptBvzBMP7t2UpLyKuAq7Emieo911BqEMnI